bentoml/_internal/frameworks/common/pytorch.py

```diff
@@ -0,0 +1,185 @@
+import pickle
+import typing as t
+import logging
+import functools
+import contextlib
+from abc import ABC
+from abc import abstractmethod
+
+from simple_di import inject
+
+from ...types import Tag
+from ...types import LazyType
+from ...utils.pkg import get_pkg_version
+from .model_runner import BaseModelRunner
+from ....exceptions import MissingDependencyException
+from ...runner.utils import Params
+from ...runner.container import Payload
+from ...runner.container import DataContainer
+from ...runner.container import DataContainerRegistry
+
+if t.TYPE_CHECKING:
+    import pytorch_lightning as pl  # disable=unused-import
+
+try:
+    import numpy as np
+    import torch
+    import torch.nn.parallel as parallel
+except ImportError:  # pragma: no cover
+    raise MissingDependencyException(
+        """
+        torch is required in order to use module `bentoml.pytorch`.
+         Instruction: Refers to https://pytorch.org/get-started/locally/
+         to setup PyTorch correctly.
+        """  # noqa
+    )
+
+_ModelType = t.Union["torch.nn.Module", "torch.jit.ScriptModule", "pl.LightningModule"]  # type: ignore[reportPrivateUsage]
```

Comment:
Lets do this under `TYPE_CHECKING`
```suggestion
ModelType = t.Union[torch.nn.Module, torch.jit.ScriptModule, pl.LightningModule]
```